Emerging_Threats/log4j.yml

```
@@ -0,0 +1,1151 @@
+version: 3
+resources:
+  replicant:
+  - exfil
+rules:
+  'Log4j #001':
+    namespace: general
+    detect:
+      events:
+      - NETWORK_CONNECTIONS
+      op: or
+      rules:
+      - op: is
+        path: event/NETWORK_ACTIVITY/?/DESTINATION/IP_ADDRESS
```

Do we only care about Destination?

I think what you want is this:
```suggestion
        path: event/NETWORK_ACTIVITY/?/IP_ADDRESS
```